src/main/java/RedBot/Helper.java

```diff
@@ -0,0 +1,32 @@
+package RedBot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Helper { //a helper class that contains QoL improvements
+    public static String[] commandParser (String[] message) { //Combines arguments in quotes
+        List<String> str = new ArrayList<String>(); //List of strings to add to
+        Boolean isInQuotes = false; //A flag to decide whether to combine or not
+        StringBuffer buffer = new StringBuffer(); //A buffer to place the strings in
+        for (String i : message) { //Iterate through the message to find any strings in quotes
+            if (message[0]==i) {continue;}
```

Comment:
It's better to use the `equals()` or `equalsIgnoreCase()` method of you're dealing with Strings.

Reply:
Will keep in mind